doc/JSON_INFO.md

```diff
@@ -92,7 +92,7 @@ Use the `Home` key to return to the top.
     - [Vehicle Groups](#vehicle-groups)
     - [Vehicle Parts](#vehicle-parts)
       - [Symbols and Variants](#symbols-and-variants)
-      - [The following optional fields are specific to CARGO or FLUIDTANK parts.](#the-following-optional-fields-are-specific-to-cargo-or-fluidtank-parts)
+      - [The following optional fields are specific to CARGO parts.](#the-following-optional-fields-are-specific-to-cargo-parts)
```

Comment:
If you are changing this line should you be changing the below?
```
#### The following optional fields are specific to CARGO or FLUIDTANK parts.
```c++
"size": 2000,                 // with flag "FLUIDTANK" this is capacity in mLs,
                              // else with "CARGO" flag the capacity in 250mL volume units.
"cargo_weight_modifier": 33,  // (Optional, default = 100) Multiplies cargo weight by this percentage.
```
```